app/Console/Commands/RedistributePromptsCommand.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
```

Comment:
@rwhite27 create a connecter name prompmine and move this there

Reply:
ok